setup-test-env.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 # Set versions
-CONTAINERD_VERSION=1.7.22
```

Comment:
This would break the logic in [line 12-14](https://github.com/runfinch/finch-daemon/blob/40d3991864516be6816abbb788118aaaf6e29802/setup-test-env.sh#L12-L14), right?

Reply:
The version will be set by env in GitHub Actions. Alternatively I can read the environment variable and default to 1.7.22 if that is more clear?

Reply:
That's fine. I missed it.